OpenKeychain/src/main/res/layout-land/backup_code_fragment.xml

```diff
@@ -1,359 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
please test in landscape-mode. I changed the arrangement of widgets in the landscape layout to fit everything despite the smaller height, moving the button to the right or something like this.